recipes/xeofs/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "xeofs" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xeofs-{{ version }}.tar.gz
+  sha256: ab3700370349dba7b4bda261471c1af01f9f00ed5bd4791e74098a5bb4537743
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - numpy >=1.19.2,<2.0.0
+    - pandas >=1.4.1,<2.0.0
+    - pooch >=1.6.0,<2.0.0
+    - python >=3.6,<4.0
```

```suggestion
    - python >=3.8
```
Ref: https://github.com/nicrie/xeofs/blob/501b330ae7905cd0ebe92b8fa3a0153fcd6755b1/pyproject.toml#L13